src/DssSpell.t.sol

```diff
@@ -887,271 +912,27 @@ contract DssSpellTest is DssSpellTestBase {
-        assertEq(WardsAbstract(address(sky)).wards(addr.addr("MKR_SKY")),        0, "before: testNewAuthorizations/mkr-sky-already-ward-sky");
-        assertEq(WardsAbstract(gov.authority()).wards(addr.addr("MKR_SKY")),     0, "before: testNewAuthorizations/mkr-sky-already-ward-mkr-authority");
-        assertEq(WardsAbstract(address(sky)).wards(addr.addr("MCD_VEST_SKY")),   0, "TestError/mcd-vest-sky-already-ward-sky");
-        assertEq(WardsAbstract(address(vat)).wards(addr.addr("SUSDS")),          0, "before: testNewAuthorizations/susds-already-ward-vat");
-        assertEq(WardsAbstract(address(flap)).wards(addr.addr("MCD_SPLIT")),     0, "before: testNewAuthorizations/splitter-already-ward-flapper");
-        assertEq(WardsAbstract(address(split)).wards(addr.addr("SPLITTER_MOM")), 0, "before: testNewAuthorizations/splitter-mom-already-ward-splitter");
-        assertEq(WardsAbstract(address(split)).wards(addr.addr("MCD_VOW")),      0, "before: testNewAuthorizations/vow-already-ward-splitter");
+    function testSkyOracleChange() public {
+        assertNotEq(flap.pip(), addr.addr("FLAP_SKY_ORACLE"), "testFlapperChange/incorrect-previous-flapper-pip-address");
+        assertEq(OsmAbstract(addr.addr("PIP_MKR")).bud(addr.addr("FLAP_SKY_ORACLE")), 0, "testNewAuthorizations/sky-oracle-already-in-mkr-oracle-buds");
 
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done(), "TestError/spell-not-done");
 
-        assertEq(WardsAbstract(address(usds)).wards(addr.addr("USDS_JOIN")),     1, "after: testNewAuthorizations/usds-join-not-ward-usds");
-        assertEq(WardsAbstract(address(sky)).wards(addr.addr("MKR_SKY")),        1, "after: testNewAuthorizations/mkr-sky-not-ward-sky");
-        assertEq(WardsAbstract(gov.authority()).wards(addr.addr("MKR_SKY")),     1, "after: testNewAuthorizations/mkr-sky-not-ward-mkr-authority");
-        assertEq(WardsAbstract(address(sky)).wards(addr.addr("MCD_VEST_SKY")),   1, "TestError/mcd-vest-sky-not-ward-sky");
-        assertEq(WardsAbstract(address(vat)).wards(addr.addr("SUSDS")),          1, "after: testNewAuthorizations/susds-not-ward-vat");
-        assertEq(WardsAbstract(address(flap)).wards(addr.addr("MCD_SPLIT")),     1, "after: testNewAuthorizations/splitter-not-ward-flapper");
-        assertEq(WardsAbstract(address(split)).wards(addr.addr("SPLITTER_MOM")), 1, "after: testNewAuthorizations/splitter-mom-not-ward-splitter");
-        assertEq(WardsAbstract(address(split)).wards(addr.addr("MCD_VOW")),      1, "after: testNewAuthorizations/vow-not-ward-splitter");
-
+        assertEq(flap.pip(), addr.addr("FLAP_SKY_ORACLE"), "testFlapperChange/incorrect-flapper-pip-address");
+        assertEq(OsmAbstract(addr.addr("PIP_MKR")).bud(addr.addr("FLAP_SKY_ORACLE")), 1, "testNewAuthorizations/sky-oracle-not-in-mkr-oracle-buds");
```

Comment:
Same, as previous comment, we should test all the params included in the exec doc: wrapper, divisor.

Reply:
- `wrapper` (or rather FLAP_SKY_ORACLE new value) is tested via `testChainlogValues` base test as well as via `testSkyOracleChange` we ensure that the address was actually changed
- `divisor` is not set anywhere, only provided for sanity check done inside init library

If you still find some coverage to be missing, please be more specific